lighthouse-core/report/report-generator.js

```diff
@@ -132,6 +132,7 @@ class ReportGenerator {
       // XSS, define a renderer that only transforms links and code snippets.
       // All other markdown ad HTML is ignored.
       const renderer = new marked.Renderer();
+      renderer.br = () => '<br>';
```

Comment:
Alright I kinda like this, hacky but with a look of intention :)
![image](https://cloud.githubusercontent.com/assets/2301202/22353242/6ab3d462-e3d4-11e6-8e56-e1db7bddf7f9.png)